kokoro/scripts/test/go_test.sh

```diff
@@ -72,8 +72,11 @@ function set_platforms() {
   # if _LOUHI_TAG_NAME is defined, set TARGET and ARCH env vars by parsing it.
   # Example value: louhi/2.46.0/shortref/windows/x86_64/start
   if [[ -n "${_LOUHI_TAG_NAME:-}" ]]; then
-    TARGET="$(echo -n "${_LOUHI_TAG_NAME}" | cut --delimiter="/" --fields=4)"
-    ARCH="$(echo -n "${_LOUHI_TAG_NAME}" | cut --delimiter="/" --fields=5)"
+    local -a _LOUHI_TAG_COMPONENTS=(${_LOUHI_TAG_NAME//\// })  
```

Comment:
I did request the opposite of igor's suggestion here, for readability and not-having-bugs-if-there-are-spaces reasons:

https://github.com/GoogleCloudPlatform/ops-agent/pull/1580#discussion_r1457914077

What's better about avoiding cut? I find the array splitting version very hard to read...

I'm going to ask for a comment if you decide to keep this version.

Reply:
I definitely prefer the current version for readability. And there should never be spaces.

What would you like the comment to say? I think the variable name and example value in the comment above cover it pretty well.

Reply:
oh i didn't see that comment, never mind I guess.